llama-index-core/llama_index/core/workflow/events.py

```diff
@@ -10,28 +10,55 @@ class Config:
         arbitrary_types_allowed = True
 
 
-class StartEvent(Event):
-    """StartEvent is implicitly sent when a workflow runs. Mimics the interface of a dict."""
+class DictLikeEvent(Event):
```

Comment:
Should we just call this `Event`? I think assuming that every event is dict-like lowers the cognitive load for the users.

Reply:
Yeah good call!